packages/base-styles/_mixins.scss

```diff
@@ -1,3 +1,83 @@
+/**
+ * Type scale mixins
+ */
+
+@mixin text-title-large {
```

Comment:
Do we want to prefix the mixins with a string e.g. `wp-components-text-title-large` to avoid potential namespace clashes?

Reply:
That seems ok to me, since there will probably be `editor` specific font styles like `editor-text-body`.